test-suite/src/test/require-jdk7/org/scalajs/testsuite/OnJDK7OrHigher.scala

```diff
@@ -0,0 +1,22 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite
+
+import org.scalajs.jasminetest.JasmineTest
+
+object OnJDK7OrHigher extends JasmineTest {
+
+  describe("require-jdk7 tests") {
+    it("should only be running JDK7 or JDK8") {
+      // This method did not exist id JDK6 and hence this class will not link
+      // java.util.Collections.emptyIterator[Int]()
+      // TODO remove comment one the implementation is added
+    }
+  }
```

Comment:
This doesn't really make any sense. I don't think we should introduce a dummy test like this.

Come to think of it, I'm not sure this commit is useful on its own, and hence doesn't qualify as a "useful unit of change". I think this should simply be squashed with the first commit that needs to put something specific to JDK 7.

Or, you could test something useful that already exists, e.g., that `java.nio.charset.StandardCharsets` constants are identical to those in `scala.scalajs.niocharset.StandardCharsets`. For JDK 8, we don't anything yet, though, I think.